### ISSUE TITLE
java.lang.NullPointerException  at scala.tools.nsc.typechecker.Typers$Typer.$anonfun$typedAnnotation$1(Typers.scala:3826)

### ISSUE THREAD

**Issue by He-Pin** [scala/bug#11312]
```
class deprecated private(message: String = "", since: String = "") extends scala.annotation.StaticAnnotation
```
and then `;clean;compile` in sbt.

```scala
[error] ## Exception when compiling 524 sources to /Users/kerr/IdeaProjects/scala/build/quick/classes/library
[error] null
[error] scala.tools.nsc.typechecker.Typers$Typer.$anonfun$typedAnnotation$1(Typers.scala:3826)
[error] scala.tools.nsc.typechecker.Typers$Typer.typedAnnotation(Typers.scala:3720)
[error] scala.tools.nsc.typechecker.Namers$Namer.$anonfun$annotSig$4(Namers.scala:1850)
[error] scala.tools.nsc.typechecker.Namers$Namer.$anonfun$annotSig$3(Namers.scala:1850)
[error] scala.reflect.internal.AnnotationInfos$LazyAnnotationInfo.forcedInfo$lzycompute(AnnotationInfos.scala:177)
[error] scala.reflect.internal.AnnotationInfos$LazyAnnotationInfo.forcedInfo(AnnotationInfos.scala:177)
[error] scala.reflect.internal.AnnotationInfos$LazyAnnotationInfo.atp(AnnotationInfos.scala:179)
[error] scala.reflect.internal.AnnotationInfos$AnnotationInfo.metaAnnotations(AnnotationInfos.scala:258)
[error] scala.reflect.internal.AnnotationInfos$AnnotationInfo$.mkFilter(AnnotationInfos.scala:138)
[error] scala.tools.nsc.typechecker.Namers$Namer$$anon$3.$anonfun$complete$9(Namers.scala:903)
[error] scala.tools.nsc.typechecker.Namers$Namer$$anon$3.complete(Namers.scala:903)
[error] scala.reflect.internal.Symbols$Symbol.info(Symbols.scala:1541)
[error] scala.reflect.internal.Symbols$Symbol.tpeHK(Symbols.scala:1476)
[error] scala.reflect.internal.Types$Type.computeMemberType(Types.scala:736)
[error] scala.reflect.internal.Symbols$MethodSymbol.typeAsMemberOf(Symbols.scala:3021)
[error] scala.reflect.internal.Types$Type.memberType(Types.scala:731)
[error] scala.tools.nsc.typechecker.Implicits$ImplicitInfo.tpe(Implicits.scala:235)
[error] scala.tools.nsc.typechecker.Implicits$ImplicitInfo.computeErroneous(Implicits.scala:255)
[error] scala.tools.nsc.typechecker.Implicits$ImplicitInfo.isCyclicOrErroneous(Implicits.scala:250)
[error] scala.tools.nsc.typechecker.Implicits$ImplicitSearch$ImplicitComputation.isIneligible(Implicits.scala:1005)
[error] scala.tools.nsc.typechecker.Implicits$ImplicitSearch$ImplicitComputation.survives(Implicits.scala:1013)
[error] scala.tools.nsc.typechecker.Implicits$ImplicitSearch$ImplicitComputation.$anonfun$eligible$2(Implicits.scala:1070)
[error] scala.tools.nsc.typechecker.Implicits$ImplicitSearch$ImplicitComputation.$anonfun$eligible$1(Implicits.scala:1070)
[error] scala.tools.nsc.typechecker.Implicits$ImplicitSearch$ImplicitComputation.<init>(Implicits.scala:1069)
[error] scala.tools.nsc.typechecker.Implicits$ImplicitSearch.searchImplicit(Implicits.scala:1207)
[error] scala.tools.nsc.typechecker.Implicits$ImplicitSearch.bestImplicit(Implicits.scala:1602)
[error] scala.tools.nsc.typechecker.Implicits.inferImplicit(Implicits.scala:104)
[error] scala.tools.nsc.typechecker.Implicits.inferImplicit$(Implicits.scala:89)
[error] scala.tools.nsc.Global$$anon$5.inferImplicit(Global.scala:477)
[error] scala.tools.nsc.typechecker.Implicits.inferImplicitFor(Implicits.scala:47)
[error] scala.tools.nsc.typechecker.Implicits.inferImplicitFor$(Implicits.scala:46)
[error] scala.tools.nsc.Global$$anon$5.inferImplicitFor(Global.scala:477)
[error] scala.tools.nsc.typechecker.Typers$Typer.applyImplicitArgs(Typers.scala:268)
[error] scala.tools.nsc.typechecker.Typers$Typer.$anonfun$adapt$2(Typers.scala:832)
[error] scala.tools.nsc.typechecker.Typers$Typer.silent(Typers.scala:704)
[error] scala.tools.nsc.typechecker.Typers$Typer.$anonfun$adapt$1(Typers.scala:831)
[error] scala.tools.nsc.typechecker.Typers$Typer.adaptToImplicitMethod$1(Typers.scala:482)
[error] scala.tools.nsc.typechecker.Typers$Typer.adapt(Typers.scala:1189)
[error] scala.tools.nsc.typechecker.Typers$Typer.typed(Typers.scala:5735)
[error] scala.tools.nsc.typechecker.Typers$Typer.typedDefDef(Typers.scala:5929)
[error] scala.tools.nsc.typechecker.Typers$Typer.typed1(Typers.scala:5630)
[error] scala.tools.nsc.typechecker.Typers$Typer.typed(Typers.scala:5721)
[error] scala.tools.nsc.typechecker.Typers$Typer.typedStat$1(Typers.scala:5785)
[error] scala.tools.nsc.typechecker.Typers$Typer.$anonfun$typedStats$10(Typers.scala:3287)
[error] scala.tools.nsc.typechecker.Typers$Typer.typedStats(Typers.scala:3287)
[error] scala.tools.nsc.typechecker.Typers$Typer.typedTemplate(Typers.scala:1952)
[error] scala.tools.nsc.typechecker.Typers$Typer.typedModuleDef(Typers.scala:1830)
[error] scala.tools.nsc.typechecker.Typers$Typer.typed1(Typers.scala:5632)
[error] scala.tools.nsc.typechecker.Typers$Typer.typed(Typers.scala:5721)
[error] scala.tools.nsc.typechecker.Typers$Typer.typedStat$1(Typers.scala:5785)
[error] scala.tools.nsc.typechecker.Typers$Typer.$anonfun$typedStats$10(Typers.scala:3287)
[error] scala.tools.nsc.typechecker.Typers$Typer.typedStats(Typers.scala:3287)
[error] scala.tools.nsc.typechecker.Typers$Typer.typedPackageDef$1(Typers.scala:5334)
[error] scala.tools.nsc.typechecker.Typers$Typer.typed1(Typers.scala:5634)
[error] scala.tools.nsc.typechecker.Typers$Typer.typed(Typers.scala:5721)
[error] scala.tools.nsc.typechecker.Analyzer$typerFactory$TyperPhase.apply(Analyzer.scala:115)
[error] scala.tools.nsc.Global$GlobalPhase.applyPhase(Global.scala:447)
[error] scala.tools.nsc.typechecker.Analyzer$typerFactory$TyperPhase.run(Analyzer.scala:104)
[error] scala.tools.nsc.Global$Run.compileUnitsInternal(Global.scala:1480)
[error] scala.tools.nsc.Global$Run.compileUnits(Global.scala:1464)
[error] scala.tools.nsc.Global$Run.compileSources(Global.scala:1456)
[error] scala.tools.nsc.Global$Run.compile(Global.scala:1580)
[error] xsbt.CachedCompiler0.run(CompilerInterface.scala:130)
[error] xsbt.CachedCompiler0.run(CompilerInterface.scala:105)
[error] xsbt.CompilerInterface.run(CompilerInterface.scala:31)
[error] sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
[error] sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[error] java.lang.reflect.Method.invoke(Method.java:498)
[error] sbt.internal.inc.AnalyzingCompiler.call(AnalyzingCompiler.scala:237)
[error] sbt.internal.inc.AnalyzingCompiler.compile(AnalyzingCompiler.scala:111)
[error] sbt.internal.inc.AnalyzingCompiler.compile(AnalyzingCompiler.scala:90)
[error] sbt.internal.inc.MixedAnalyzingCompiler.$anonfun$compile$3(MixedAnalyzingCompiler.scala:82)
[error] scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
[error] sbt.internal.inc.MixedAnalyzingCompiler.timed(MixedAnalyzingCompiler.scala:133)
[error] sbt.internal.inc.MixedAnalyzingCompiler.compileScala$1(MixedAnalyzingCompiler.scala:73)
[error] sbt.internal.inc.MixedAnalyzingCompiler.compile(MixedAnalyzingCompiler.scala:114)
[error] sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileInternal$1(IncrementalCompilerImpl.scala:307)
[error] sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileInternal$1$adapted(IncrementalCompilerImpl.scala:307)
[error] sbt.internal.inc.Incremental$.doCompile(Incremental.scala:106)
[error] sbt.internal.inc.Incremental$.$anonfun$compile$4(Incremental.scala:87)
[error] sbt.internal.inc.IncrementalCommon.recompileClasses(IncrementalCommon.scala:116)
[error] sbt.internal.inc.IncrementalCommon.cycle(IncrementalCommon.scala:63)
[error] sbt.internal.inc.Incremental$.$anonfun$compile$3(Incremental.scala:89)
[error] sbt.internal.inc.Incremental$.manageClassfiles(Incremental.scala:134)
[error] sbt.internal.inc.Incremental$.compile(Incremental.scala:80)
[error] sbt.internal.inc.IncrementalCompile$.apply(Compile.scala:67)
[error] sbt.internal.inc.IncrementalCompilerImpl.compileInternal(IncrementalCompilerImpl.scala:311)
[error] sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileIncrementally$1(IncrementalCompilerImpl.scala:269)
[error] sbt.internal.inc.IncrementalCompilerImpl.handleCompilationError(IncrementalCompilerImpl.scala:159)
[error] sbt.internal.inc.IncrementalCompilerImpl.compileIncrementally(IncrementalCompilerImpl.scala:238)
[error] sbt.internal.inc.IncrementalCompilerImpl.compile(IncrementalCompilerImpl.scala:69)
[error] sbt.Defaults$.compileIncrementalTaskImpl(Defaults.scala:1549)
[error] sbt.Defaults$.$anonfun$compileIncrementalTask$1(Defaults.scala:1523)
[error] scala.Function1.$anonfun$compose$1(Function1.scala:44)
[error] sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:40)
[error] sbt.std.Transform$$anon$4.work(System.scala:67)
[error] sbt.Execute.$anonfun$submit$2(Execute.scala:269)
[error] sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[error] sbt.Execute.work(Execute.scala:278)
[error] sbt.Execute.$anonfun$submit$1(Execute.scala:269)
[error] sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
[error] sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
[error] java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error] java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error] java.lang.Thread.run(Thread.java:748)
[error]            
[error] java.lang.NullPointerException
[error]         at scala.tools.nsc.typechecker.Typers$Typer.$anonfun$typedAnnotation$1(Typers.scala:3826)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.typedAnnotation(Typers.scala:3720)
[error]         at scala.tools.nsc.typechecker.Namers$Namer.$anonfun$annotSig$4(Namers.scala:1850)
[error]         at scala.tools.nsc.typechecker.Namers$Namer.$anonfun$annotSig$3(Namers.scala:1850)
[error]         at scala.reflect.internal.AnnotationInfos$LazyAnnotationInfo.forcedInfo$lzycompute(AnnotationInfos.scala:177)
[error]         at scala.reflect.internal.AnnotationInfos$LazyAnnotationInfo.forcedInfo(AnnotationInfos.scala:177)
[error]         at scala.reflect.internal.AnnotationInfos$LazyAnnotationInfo.atp(AnnotationInfos.scala:179)
[error]         at scala.reflect.internal.AnnotationInfos$AnnotationInfo.metaAnnotations(AnnotationInfos.scala:258)
[error]         at scala.reflect.internal.AnnotationInfos$AnnotationInfo$.mkFilter(AnnotationInfos.scala:138)
[error]         at scala.tools.nsc.typechecker.Namers$Namer$$anon$3.$anonfun$complete$9(Namers.scala:903)
[error]         at scala.tools.nsc.typechecker.Namers$Namer$$anon$3.complete(Namers.scala:903)
[error]         at scala.reflect.internal.Symbols$Symbol.info(Symbols.scala:1541)
[error]         at scala.reflect.internal.Symbols$Symbol.tpeHK(Symbols.scala:1476)
[error]         at scala.reflect.internal.Types$Type.computeMemberType(Types.scala:736)
[error]         at scala.reflect.internal.Symbols$MethodSymbol.typeAsMemberOf(Symbols.scala:3021)
[error]         at scala.reflect.internal.Types$Type.memberType(Types.scala:731)
[error]         at scala.tools.nsc.typechecker.Implicits$ImplicitInfo.tpe(Implicits.scala:235)
[error]         at scala.tools.nsc.typechecker.Implicits$ImplicitInfo.computeErroneous(Implicits.scala:255)
[error]         at scala.tools.nsc.typechecker.Implicits$ImplicitInfo.isCyclicOrErroneous(Implicits.scala:250)
[error]         at scala.tools.nsc.typechecker.Implicits$ImplicitSearch$ImplicitComputation.isIneligible(Implicits.scala:1005)
[error]         at scala.tools.nsc.typechecker.Implicits$ImplicitSearch$ImplicitComputation.survives(Implicits.scala:1013)
[error]         at scala.tools.nsc.typechecker.Implicits$ImplicitSearch$ImplicitComputation.$anonfun$eligible$2(Implicits.scala:1070)
[error]         at scala.tools.nsc.typechecker.Implicits$ImplicitSearch$ImplicitComputation.$anonfun$eligible$1(Implicits.scala:1070)
[error]         at scala.tools.nsc.typechecker.Implicits$ImplicitSearch$ImplicitComputation.<init>(Implicits.scala:1069)
[error]         at scala.tools.nsc.typechecker.Implicits$ImplicitSearch.searchImplicit(Implicits.scala:1207)
[error]         at scala.tools.nsc.typechecker.Implicits$ImplicitSearch.bestImplicit(Implicits.scala:1602)
[error]         at scala.tools.nsc.typechecker.Implicits.inferImplicit(Implicits.scala:104)
[error]         at scala.tools.nsc.typechecker.Implicits.inferImplicit$(Implicits.scala:89)
[error]         at scala.tools.nsc.Global$$anon$5.inferImplicit(Global.scala:477)
[error]         at scala.tools.nsc.typechecker.Implicits.inferImplicitFor(Implicits.scala:47)
[error]         at scala.tools.nsc.typechecker.Implicits.inferImplicitFor$(Implicits.scala:46)
[error]         at scala.tools.nsc.Global$$anon$5.inferImplicitFor(Global.scala:477)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.applyImplicitArgs(Typers.scala:268)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.$anonfun$adapt$2(Typers.scala:832)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.silent(Typers.scala:704)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.$anonfun$adapt$1(Typers.scala:831)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.adaptToImplicitMethod$1(Typers.scala:482)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.adapt(Typers.scala:1189)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.typed(Typers.scala:5735)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.typedDefDef(Typers.scala:5929)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.typed1(Typers.scala:5630)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.typed(Typers.scala:5721)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.typedStat$1(Typers.scala:5785)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.$anonfun$typedStats$10(Typers.scala:3287)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.typedStats(Typers.scala:3287)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.typedTemplate(Typers.scala:1952)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.typedModuleDef(Typers.scala:1830)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.typed1(Typers.scala:5632)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.typed(Typers.scala:5721)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.typedStat$1(Typers.scala:5785)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.$anonfun$typedStats$10(Typers.scala:3287)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.typedStats(Typers.scala:3287)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.typedPackageDef$1(Typers.scala:5334)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.typed1(Typers.scala:5634)
[error]         at scala.tools.nsc.typechecker.Typers$Typer.typed(Typers.scala:5721)
[error]         at scala.tools.nsc.typechecker.Analyzer$typerFactory$TyperPhase.apply(Analyzer.scala:115)
[error]         at scala.tools.nsc.Global$GlobalPhase.applyPhase(Global.scala:447)
[error]         at scala.tools.nsc.typechecker.Analyzer$typerFactory$TyperPhase.run(Analyzer.scala:104)
[error]         at scala.tools.nsc.Global$Run.compileUnitsInternal(Global.scala:1480)
[error]         at scala.tools.nsc.Global$Run.compileUnits(Global.scala:1464)
[error]         at scala.tools.nsc.Global$Run.compileSources(Global.scala:1456)
[error]         at scala.tools.nsc.Global$Run.compile(Global.scala:1580)
[error]         at xsbt.CachedCompiler0.run(CompilerInterface.scala:130)
[error]         at xsbt.CachedCompiler0.run(CompilerInterface.scala:105)
[error]         at xsbt.CompilerInterface.run(CompilerInterface.scala:31)
[error]         at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
[error]         at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[error]         at java.lang.reflect.Method.invoke(Method.java:498)
[error]         at sbt.internal.inc.AnalyzingCompiler.call(AnalyzingCompiler.scala:237)
[error]         at sbt.internal.inc.AnalyzingCompiler.compile(AnalyzingCompiler.scala:111)
[error]         at sbt.internal.inc.AnalyzingCompiler.compile(AnalyzingCompiler.scala:90)
[error]         at sbt.internal.inc.MixedAnalyzingCompiler.$anonfun$compile$3(MixedAnalyzingCompiler.scala:82)
[error]         at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
[error]         at sbt.internal.inc.MixedAnalyzingCompiler.timed(MixedAnalyzingCompiler.scala:133)
[error]         at sbt.internal.inc.MixedAnalyzingCompiler.compileScala$1(MixedAnalyzingCompiler.scala:73)
[error]         at sbt.internal.inc.MixedAnalyzingCompiler.compile(MixedAnalyzingCompiler.scala:114)
[error]         at sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileInternal$1(IncrementalCompilerImpl.scala:307)
[error]         at sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileInternal$1$adapted(IncrementalCompilerImpl.scala:307)
[error]         at sbt.internal.inc.Incremental$.doCompile(Incremental.scala:106)
[error]         at sbt.internal.inc.Incremental$.$anonfun$compile$4(Incremental.scala:87)
[error]         at sbt.internal.inc.IncrementalCommon.recompileClasses(IncrementalCommon.scala:116)
[error]         at sbt.internal.inc.IncrementalCommon.cycle(IncrementalCommon.scala:63)
[error]         at sbt.internal.inc.Incremental$.$anonfun$compile$3(Incremental.scala:89)
[error]         at sbt.internal.inc.Incremental$.manageClassfiles(Incremental.scala:134)
[error]         at sbt.internal.inc.Incremental$.compile(Incremental.scala:80)
[error]         at sbt.internal.inc.IncrementalCompile$.apply(Compile.scala:67)
[error]         at sbt.internal.inc.IncrementalCompilerImpl.compileInternal(IncrementalCompilerImpl.scala:311)
[error]         at sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileIncrementally$1(IncrementalCompilerImpl.scala:269)
[error]         at sbt.internal.inc.IncrementalCompilerImpl.handleCompilationError(IncrementalCompilerImpl.scala:159)
[error]         at sbt.internal.inc.IncrementalCompilerImpl.compileIncrementally(IncrementalCompilerImpl.scala:238)
[error]         at sbt.internal.inc.IncrementalCompilerImpl.compile(IncrementalCompilerImpl.scala:69)
[error]         at sbt.Defaults$.compileIncrementalTaskImpl(Defaults.scala:1549)
[error]         at sbt.Defaults$.$anonfun$compileIncrementalTask$1(Defaults.scala:1523)
[error]         at scala.Function1.$anonfun$compose$1(Function1.scala:44)
[error]         at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:40)
[error]         at sbt.std.Transform$$anon$4.work(System.scala:67)
[error]         at sbt.Execute.$anonfun$submit$2(Execute.scala:269)
[error]         at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[error]         at sbt.Execute.work(Execute.scala:278)
[error]         at sbt.Execute.$anonfun$submit$1(Execute.scala:269)
[error]         at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
[error]         at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
[error]         at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error]         at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error]         at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error]         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error]         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error]         at java.lang.Thread.run(Thread.java:748)
[error] (library / Compile / compileIncremental) java.lang.NullPointerException
[error] Total time: 2 s, completed 2018-12-14 20:08:40

```

**Comment by kajmagnus** [comment]
**(** This doesn't happen to be caused by the stack overflow issue here: https://github.com/sbt/sbt/issues/4182 ?

I was getting this error:

```
app_1       | [error] ## Exception when compiling 36 sources to /opt/talkyard/app/modules/ed-core/target/scala-2.12/classes
app_1       | [error] null
app_1       | [error] scala.tools.nsc.transform.Erasure$Eraser.adaptMember(Erasure.scala:709)
app_1       | [error] scala.tools.nsc.transform.Erasure$Eraser.typed1(Erasure.scala:796)
app_1       | [error] scala.tools.nsc.typechecker.Typers$Typer.typed(Typers.scala:5726)
app_1       | [error] scala.tools.nsc.typechecker.Typers$Typer.$anonfun$typed1$38(Typers.scala:4855)
app_1       | [error] scala.tools.nsc.typechecker.Typers$Typer.silent(Typers.scala:713)
app_1       | [error] scala.tools.nsc.typechecker.Typers$Typer.normalTypedApply$1(Typers.scala:4857)
app_1       | [error] scala.tools.nsc.typechecker.Typers$Typer.typedApply$1(Typers.scala:4885)
app_1       | [error] scala.tools.nsc.typechecker.Typers$Typer.typed1(Typers.scala:5680)
app_1       | [error] scala.tools.nsc.transform.Erasure$Eraser.typed1(Erasure.scala:796)
app_1       | [error] scala.tools.nsc.typechecker.Typers$Typer.typed(Typers.scala:5726)
...
```

and turns out it was caused by that stack overflow problem, and increasing the stack size: `-Xss5m` solved that problem, in my case.  **)**

**Comment by som-snytt** [comment]
There is no information here about which version of Scala and Intellij/sbt or what is compiled besides the annotation, which cannot be used because its constructor is private. Stack trace says implicit search forces the annotation.

**Comment by SethTisue** [comment]
It's not clear to me how to reproduce this. Happy to reopen if those details are added.

**Comment by som-snytt** [comment]
oh, I guess this is just "clean compile the scala project after making the constructor private". It fails correctly now, "constructor deprecated in class deprecated cannot be accessed in trait App from trait App in package scala".

contemporaneous 
https://github.com/scala/scala/pull/7449
https://github.com/scala/scala/pull/7531

oh actually it does fails on a fresh checkout of 2.13.0. How about that. That's after a couple of "cannot be accessed" errors, but scalac doesn't provided any tree context on NPE here.

**Comment by som-snytt** [comment]
The crash is fixed at 2.13.2, with starr at 2.13.1. It looks like the crash was in the deprecation annotation of `any2stringadd`.
